diff --git a/dist/pluralize.js b/dist/pluralize.js
index 3eccf12f8d656480d6ac4c107d8981d5566099d6..644ccf3098274d2ea0399a5d7390f8143141abb8 100644
--- a/dist/pluralize.js
+++ b/dist/pluralize.js
@@ -123,7 +123,7 @@ function checkWord(replaceMap, keepMap, rules) {
  */
 const pluralize = function (word, count, inclusive) {
     const pluralized = count === 1 ? pluralize.singular(word) : pluralize.plural(word);
-    return (inclusive ? count + ' ' : '') + pluralized;
+    return (inclusive && count ? count.toLocaleString() + ' ' : '') + pluralized;
 };
 /**
  * Returns the plural form of a word.
